openapi/index.yaml

```diff
@@ -211,7 +211,7 @@ definitions:
   FamilyUID:
     description: An hash code that identifies univocally a family
     type: string
-    minLength: 16
+    minLength: 1
```

Comment:
isn't this a sha256 hash ? (64 char length)

Reply:
it won't work because a bug in `io-utils` - better leave it as is for now